atd-vzv/src/views/map/AnimatedIcon.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { CanvasOverlay } from "react-map-gl";
+
+const AnimatedIcon = ({ location, paint }) => {
+  const { x, y } = location;
+  const _redraw = ({ width, height, ctx, project }) => {
+    const coordinates = project([x, y]);
+
+    var size = 66.6667;
+    var duration = 1000;
+    var t = (performance.now() % duration) / duration;
+
+    var radius = (size / 2) * 0.3;
+    var outerRadius = (size / 2) * 0.7 * t + radius;
+
```

Comment:
We do not use var, my linter is throwing warnings. For variables, we use either `const` for that will not change, and `let` for values that will change. These all seem constants, it would be great help to us if these can be made consts so that we can be consistent with our conventions.

Reply:
Yes I will change them to `const`s!